django/core/checks/templates.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+
+from . import Error, Tags, register
+
+E001 = Error(
+    "You have APP_DIRS = True in your TEMPLATES, while explicitly specifying"
+    "the template loaders. Either set APP_DIRS to False or remove the "
```

Comment:
I thinking removing APP_DIRS from TEMPLATES (since it defaults to False) is a better suggestion than setting it to False.